bin/client_run

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 
-sudo ./build/client/mysterium_client --config-dir=bin/client_package/config --runtime-dir=build/client $@
+source bin/helpers/load_run_environment.sh
+
+sudo ./build/client/mysterium_client \
+   --config-dir=bin/client_package/config \
+   --runtime-dir=build/client \
+   $DISCOVERY \
```

Comment:
Shouldn't this param be passed in in docker-entrypoints as well?

Reply:
it depends from environment we want to build for.. we should compile in for testnet and only for development to show our own discovery.

Reply:
is is passed now